src/styles/reset.css

```diff
@@ -0,0 +1,4 @@
+h2,
```

Comment:
I think you could just have reset.scss file

Reply:
I deleted redudant css file.